bin/p2-launch/main.go

```diff
@@ -36,3 +54,73 @@ func main() {
 		log.Fatalln("Unsuccessful launch of one or more things in the manifest")
 	}
 }
+
+func authorize(manifest pods.Manifest) error {
```

Comment:
it is, but I don't know what I can put in `auth` that isn't some weird function like `func GetAuthPolicy(authType string, keyringPath *string, deployPolicy *string)` and callers would be providing `nil` values all around